src/libraries/System.Diagnostics.PerformanceCounter/src/System/Diagnostics/SharedPerformanceCounter.cs

```diff
@@ -1756,7 +1761,7 @@ private unsafe void Initialize(string fileMappingName, int fileMappingSize, int
                 }
                 finally
                 {
-                    securityDescriptorPointer?.Close();
+                    Interop.Kernel32.LocalFree((IntPtr)pSecurityDescriptor);
```

Comment:
SafeLocalAllocHandle was using Marshal.FreeHGlobal.  Let's keep using it here.

Reply:
`pSecurityDescriptor` is assigned by `ConvertStringSecurityDescriptorToSecurityDescriptor`, for which documentation states:
> To free the returned buffer, call the [LocalFree](https://learn.microsoft.com/en-us/windows/desktop/api/winbase/nf-winbase-localfree) function.

If there's a reason to prefer an indirect call through `Marshal.FreeHGlobal`, I'll make the changes.

Reply:
Simply that a) it's what was being used before and b) we don't need to add another P/Invoke to this library and can just call the public API.

Reply:
Done.